runners/flink/src/main/java/org/apache/beam/runners/flink/FlinkBatchTransformTranslators.java

```diff
@@ -396,12 +396,8 @@ public void translateNode(
       // construct a map from side input to WindowingStrategy so that
       // the DoFn runner can map main-input windows to side input windows
       Map<PCollectionView<?>, WindowingStrategy<?, ?>> sideInputStrategies = new HashMap<>();
-      List<PCollectionView<?>> sideInputs;
-      try {
-        sideInputs = CombineTranslation.getSideInputs(context.getCurrentTransform());
-      } catch (IOException e) {
-        throw new RuntimeException(e);
-      }
+      List<PCollectionView<?>> sideInputs = new ArrayList<>();
```

Comment:
Since we no longer have side inputs in Combine, can you go ahead and remove all references to side inputs in this function?

Reply:
Removed all references in this function and left TODOs in FlinkReduceFunction and FlinkPartialReduceFunction.